tests/km_core_tests.bats

```diff
@@ -1713,14 +1713,14 @@ EOF
 
    # wait for the km mgmt pipe to exist
    tries=10
-   for ((i=0; i<10; i++))
+   while [ $tries -gt 0 ]
```

Comment:
**for i in {1..10}; do done** is bash way.

Reply:
I want to see how many tries are remaining when we exit the loop.
If $tries > 0 then the test succeed.  If $tries == 0 then the test failed.

I want the value of $tries to have a scope outside the loop.
I don't know what the scope of $i is in your example and I suspect others don't know either.
What I have is pretty clear as to the scope of $tries.